webapp/channels/src/components/profile_popover/profile_popover_email.tsx

```diff
@@ -4,9 +4,9 @@
 import React from 'react';
 
 type Props = {
-    email: string;
-    haveOverrideProp: boolean;
-    isBot: boolean;
+    email?: string;
+    haveOverrideProp?: boolean;
+    isBot?: boolean;
```

Comment:
Why make these all optional? I would've assumed that we should have them by the time that we're rendering the contents of the ProfilePopover

Reply:
This should have been optional in the beginning because the components has a null check for all these before